src/sass/components/_section--promotion.scss

```diff
@@ -0,0 +1,255 @@
+.section--promotion {
+  padding: 0 0 1.875rem 0;
+
+  .promotion {
+    height: 530px;
+    padding: 0 30px;
+
+    .top-bar {
+      background: $text-color;
+      height: 50px;
+      color: white;
+      text-transform: uppercase;
+      font-size: 24px;
+      font-weight: 500;
+      padding: 7px 30px;
+      position: relative;
+
+      .dots {
+        position: relative;
+        right: -120px;
+
+        ul {
+          margin: 0;
+          padding: 0 0 0 1rem;
+          list-style: none;
+          width: 90px;
+          transform: translateY(15px);
+          left: 90px;
+          bottom: 20px;
+
+          li {
+            width: 15px;
+            height: 15px;
+            background: $promotion-dots-color-grey;
+            margin: 4px;
+            border-radius: 50%;
+
+            &:hover,
+            &.active {
+              background: $primary;
+            }
+          }
+        }
+      }
+    }
+
+    .photo {
+      background-size: cover;
+      background-position: center;
+      height: 300px;
+
+      .hover {
+        opacity: 0;
+        height: 100%;
+        transition: .3s;
+
+        &:hover {
+          opacity: 1;
+        }
+
+        .btn-main {
+          margin-top: 100px;
+          text-transform: uppercase;
+          font-weight: 400;
+          border: none;
+        }
+
+        .counter {
+          margin-top: 40px;
+
+          .time {
+            display: inline-block;
+            width: 70px;
+            height: 70px;
+            background: rgba(42,42,42,0.8);
+            border-radius: 50%;
+            margin: 2px;
+            color: white;
+            text-transform: uppercase;
+            padding-top: 15px;
+            font-size: 1.5rem;
+            line-height: 1.5rem;
+
+            span {
+              margin-top: 5px;
+              display: block;
+              font-size: 0.9rem;
+              line-height: 8px;
+            }
+          }
+        }
+
+      }
+    }
+
+    .photo1 {
+      background-image: url(../images/promotion/sofa.jpg);
+    }
+
+    .photo2 {
+      background-image: url(../images/promotion/baby.jpg);
+    }
+
+    .photo3 {
+      background-image: url(../images/promotion/room.jpg);
+    }
+
+    .bottom-bar {
+
+      p {
+        font-weight: 700;
+        font-size: 1.2em;
+        margin: 10px 0;
+        text-align: center;
+      }
+
+      .stars {
+        text-align: center;
+
+        a {
+          text-decoration: none;
+          color: $text-color;
+          display: inline-block;
+          font-style: normal;
+          font-variant: normal;
+          text-rendering: auto;
+          line-height: 1;
+          font-family: 'Font Awesome 5 Free';
+          font-weight: 400;
+
+          &.full {
+            font-weight: 900;
+          }
+          &:before {
+            content: "\f005";
```

Comment:
why don't you use classes from `Font Awesome` directly?

Reply:
I agree - it's much easier to understand a class in HTML (let's say `fa-facebook`) instead of `content: "\xxxx"` in CSS.